tests/tests_C2_economic_functions.py

```diff
@@ -32,54 +35,66 @@ def test_crf():
     assert CRF == (0.1 * (1 + 0.1) ** 20) / ((1 + 0.1) ** 20 - 1)
```

Comment:
```suggestion
    assert CRF == crf
```
?

Reply:
but for instance in this one, it's different as here we are trying out the equation of the CRF.
In the previous example, CRF is a part of the equation we are asserting. Does that make sense?

Reply:
Okay think you get what you mean. still the equation is made up of some variables that could be defined at the top: discount factor D, project lifetime T and tax. Better to use the definitions above them to write out the numbers directly in the assertion.